CONTRIBUTING.rst

```diff
@@ -18,12 +18,72 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, PEPs that are not marked as Draft, Active or Provisional are
+considered to be historical documents rather than living specifications
+or documentation. Major changes to their core content usually require
+a new PEP, while smaller modifications may or may not be appropriate,
+depending on the PEP's status:
+
+* **Rejected**, **Deferred**, **Withdrawn** and **Superceded** PEPs should
+  usually only be modified when necessary to correct technical issues with
+  header processing or that prevent content from rendering as originally
+  intended. Occasionally, a Deferred or even Withdrawn PEP may be resurrected
+  with major updates, but it is often better to just propose a new one.
+
+* **Final** PEPs are considered complete; while small fixes to clearly
+  unintentional errors in the document that meaningfully impair understanding
+  may be considered, substantive changes to the content generally will not.
+
+* **Accepted** PEPs may be updated to reflect minor details of the final
+  implementation, but significant changes to the normative specification
```

Comment:
Is there a plain English synonym for "normative" that can be used? I had to look it up :)

Reply:
We can just remove "normative".

Reply:
I just removed it; its an extra word and isn't critical to meaning here.

EDIT: Seems like I forgot to both actually push my changes from many hours ago, and submit this reply/resolve this comment. Oops, my fault.